lib/demo_web/templates/layout/app.html.eex

```diff
@@ -12,13 +12,8 @@
     <header>
       <section class="container">
         <nav role="navigation">
-          <ul>
-            <li><a href="https://hexdocs.pm/phoenix/overview.html">Get Started</a></li>
-          </ul>
+          <%= render "_user_menu.html", assigns %>
```

Comment:
Note how we generate the menu as a separate file. We can probably add this render automatically to the app layout (or root layout if using LV) after the `<body> `tag.